### ISSUE TITLE
Short circuit warning when resolutionType: 'parallel'

### ISSUE THREAD

**Issue by robrechtme** [platypusrex/next-merge-props#16]
Hi

After upgrading to v1 I get a warning 
```
🟠 Short circuit is not supported for parallel resolution
```
even though I haven't defined a short circuit option: 
```tsx
export const getStaticProps = mergeProps([loaderA(), loaderB()], {
  resolutionType: 'parallel',
});
```

I believe this warning should only be shown when the `shortCircuit` option is passed along with   `resolutionType: 'parallel'`, no?

**Comment by platypusrex** [comment]
Thanks @robrechtme and nice catch. I'll release a patch for this shortly 👍

**Comment by platypusrex** [comment]
@robrechtme published new patch version (1.0.1). Resolved in #17.
